Utilities/PlayerUtils.cs

```diff
@@ -889,6 +889,23 @@ public static void HideAccessories(this Player player, bool hideHeadAccs = true,
         /// <returns>Returns <see langword="true"/> if the player CAN'T use the item.</returns>
         public static bool CantUseHoldout(this Player player, bool needsToHold = true) => player == null || !player.active || player.dead || (!player.channel && needsToHold) || player.CCed || player.noItems;
 
+        /// <summary>
+        /// A short method that heals the player.
+        /// All direct heals in Calamity should use this.
+        /// </summary>
+        /// <param name="player">The player being healed.</param>
+        /// <param name="amount">The amount of life being healed.</param>
+        /// <param name="healEffect">Whether the heal CombatText should be displayed.</param>
+        /// <param name="broadcast">Whether the heal CombatText should be synced.</param>
+        public static void HealPlayer(this Player player, int amount, bool healEffect = true, bool broadcast = true)
+        {
+            player.statLife += amount;
+            if (player.statLife > player.statLifeMax2)
+                player.statLife = player.statLifeMax2;
+            if (healEffect)
+                player.HealEffect(amount, broadcast);
```

Comment:
Consideration:
- Instead of making two bool flags, We probably could make `HealTextType` enum and That would be lot more cleaner to read

Example code:
```cs
enum HealTextType
{
  None,
  Local,
  Broadcast
}

player.HealPlayer(100, HealTextType.Broadcast);
```

Reply:
I considered the enum approach aswell, but decided against it to more closely mirror the old method being replaced.

Ultimately it’s a preferential thing, and if you Calamity devs prefer it that way, I’ll quickly change it no problem.

Reply:
I agree with swapping it to an enum. It's a trinary state, so using two bools will always have an erroneous fourth state (heal effect false with broadcast true, in this case).